tests/internal/aws_s3_local_buffer.c

```diff
@@ -0,0 +1,154 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+#include <fluent-bit/flb_sds.h>
+#include <fluent-bit/flb_s3_local_buffer.h>
+#include <fluent-bit/flb_output_plugin.h>
+#include <fluent-bit/flb_mem.h>
+#include <fluent-bit/flb_info.h>
+#include <fluent-bit/flb_aws_util.h>
+
+#include "flb_tests_internal.h"
+
+#define DIRECTORY "/fluent-bit/buffer/s3"
+#define PLUGIN_NAME "flb_s3_plugin"
+#define TEST_DATA "I love Fluent Bit"
+#define KEY_1 "key1"
+#define KEY_2 "key2"
+#define KEY_3 "key3"
+
+static void test_chunk_compare(struct flb_local_buffer *store, 
```

Comment:
How about just "check_chunk"? (Just a suggestion)- "test_chunk_compare" sounds to me like it would compare two chunks.

Reply:
Done